README.md

```diff
@@ -44,57 +44,43 @@ For building and deploying the software Gradle was in use with the following plu
 ![spring](./doc/img/spring.png) 
 
 #### Spring Boot
-Spring Boot helps with the creation of stand-alone Spring based applications.
+Spring boot allows for automatic dependency injection implemented with classpath scanning instead of
+using xml configuration for the dependency injection.
 
-#### Spring Data
+#### Spring Data Rest
 The rapid api development architecture provided by spring-data-rest was used to develop the api.
 It provides an easy way to build REST web services on top of the data repositories. The
-PagingAndSortingRepositories provide request handling, deserialization, crud on the database,
-serialization and rendering of errors. Validation was implemented with Bean validations and if
-necessary with Spring event handlers. Side effects were also done with Spring event handlers.
+PagingAndSortingRepositories and the corresponding entities provide everything for CRUD operations on the database
+(Deserialization, Database Operation, Serialization, Basic Error Handling).
+Validation was implemented with Bean validations and if necessary with Spring event handlers.
+Side effects were also done with Spring event handlers.
 
 ### Hibernate
 ![hibernate](./doc/img/hibernate.png)
 
-The data was stored in a hibernate database which offers an object relational mapping of the data.
-This means object oriented entities can be stored in a relational database.
-
-### JPA
-Jakarta Persistence API is the API for storing, retrieving, updating and deleting relational database
-entries in an object oriented context.
-
-### Rapid API
-![rapid api](./doc/img/rapid_api.png)
-
-??? --> Frontend?
+The data was stored in a h2 database. The data is accessed through the Object Relational Mapper hibernate.
 
 ### PlantUML
 ![plant](./doc/img/plant_uml.png)
 
-UML diagrams were created with PlantUML plain text language.
-
-### Git, Github
-The versioning of the source code and other files was maintained by Git ont the GitHub platform.
-The GitHubs Actions feature was used for automated build, test and deployment of the software.
+UML diagrams were created with PlantUML.
 
 ## High-level components
 ### Database
-All entities used for the game are persisted during a running game. The main reason is that Spring Data REST
```

Comment:
The main reason for using a database is that it provides persistence.
The other stuff is done with the magic of spring data rest. But this has nothing to do with the database.